preprocess/clean/CA/.gitignore

```diff
@@ -0,0 +1,2 @@
+input
+output
```

Comment:
I think it'd be better to put this in the main `.gitignore` so that we don't have to chase down rules throughout the codebase.